script.js

```diff
@@ -1,4 +1,95 @@
+//set global variables
```

Comment:
Good use of comments here and throughout the code to describe what the various lines of code are for